Meshtastic/Helpers/BLEManager.swift

```diff
@@ -883,12 +890,13 @@ class BLEManager: NSObject, CBPeripheralDelegate, MqttClientProxyManagerDelegate
 								}
 							}
 							hopNodes.append(traceRouteHop)
-							routeString += "\(hopNode?.user?.longName ?? (node == 4294967295 ? "Repeater" : String(hopNode?.num.toHex() ?? "unknown".localized))) \(hopNode?.viaMqtt ?? false ? "MQTT" : "") (\(traceRouteHop.snr > 0 ? hopNode?.snr ?? 0.0 : 0.0)dB) --> "
+							routeString += "\(hopNode?.user?.longName ?? (node == 4294967295 ? "Repeater" : String(hopNode?.num.toHex() ?? "unknown".localized))) \(hopNode?.viaMqtt ?? false ? "MQTT" : "") (\(traceRouteHop.snr != -32 ? String(traceRouteHop.snr) : "unknown ".localized)dB) --> "
```

Comment:
One more thing: "Repeater" is not really accurate, it can also just be a node that doesn't have the encryption key, or for the route back it can be old firmware.

Reply:
In that case, would it be reasonable to just drop the "node == 4294967295" handling and let it fall through to the remaining options? It would try to use the node number or fall back to unknown.

Reply:
Yes, sounds reasonable to me.

Reply:
Done.